flight_phase_files/HALO_RF02_20200122_info.yaml

```
@@ -24,59 +24,125 @@ remarks:
 - Descending Cirrus (9-13.5km) coming from West and South of the circle.
 - Away from the southwest edge of the track the air was almost entirely cloud free.
 segments:
-- kind: circle
-  name: c1
-  start: 2020-01-22 15:18:17
-  end: 2020-01-22 16:13:02
-- kind: circle_break
-  name: cb1to2
-  start: 2020-01-22 16:13:02
-  end: 2020-01-22 16:31:19
-- kind: circle
-  name: c2
-  start: 2020-01-22 16:31:19
-  end: 2020-01-22 17:26:29
-- kind: circle_break
-  name: cb2to3
-  start: 2020-01-22 17:26:29
-  end: 2020-01-22 17:43:08
-- kind: circle
-  name: c3
-  start: 2020-01-22 17:43:08
-  end: 2020-01-22 18:38:02
-- kind: baccardi_calibration
-  name: baccardi_cal
+- kinds:
+  - circle
+  name: Circle 1
+  irregularities:
+  - Circle is  363.52 deg to accomodate all 12 dropsondes
+  segment_id: HALO-0122_c1
+  start: 2020-01-22 15:17:17
```

The circle does not start 1 minute before the first launch
```suggestion
  start: 2020-01-22 15:17:18
```

---

resolved with the other comments